ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue

```diff
@@ -123,9 +131,10 @@ export default defineComponent({
 
     const localState = reactive({
       currentlyEditingHomeOwnerId: -1,
-      isEditing: computed((): boolean => {
-        return localState.currentlyEditingHomeOwnerId >= 0
-      })
+      isEditingMode: computed(
+        (): boolean => localState.currentlyEditingHomeOwnerId >= 0
+      ),
```

Comment:
I know it's all the same but perhaps we preserve the formatting of the computed values the same as done elsewhere.

Reply:
It's the prettier that formats it like this...

Reply:
ahh fair enough :) 
100% All the same for sure, just nice to follow the previously done stuff for consistency